mongodb/src/test/java/org/hibernate/ogm/datastore/mongodb/test/options/associationstorage/AssociationStorageOptionTest.java

```diff
@@ -33,16 +33,17 @@ public void testAssociationStorageMappingOption() throws Exception {
 		ConfigurationContext configurationContext = new ConfigurationContextImpl( context );
 
 		new MongoDB().getConfigurationBuilder( configurationContext )
-			.entity( ExampleForMongoDBMapping.class )
+				.entity( ExampleForMongoDBMapping.class )
 				.property( "content", ElementType.FIELD )
-					.associationStorage( AssociationStorageType.ASSOCIATION_DOCUMENT );
+				.associationStorage( AssociationStorageType.ASSOCIATION_DOCUMENT );
```

Comment:
The previous indentation had a specific meaning. Please be careful in not throwing such things away.